packages/cli/odahuflow/cli/parsers/local/packaging.py

```diff
@@ -240,10 +319,15 @@ def run(client: ModelPackagingClient, pack_id: str, manifest_file: List[str], ma
 
     if disable_target:
         LOGGER.debug(f'Next targets are disabled: {", ".join(disable_target)}')
-    targets = get_packager_targets(
-        [t for t in mp.spec.targets if t.name not in disable_target and not is_target_disabled],
-        connections, ConnectionClient.construct_from_other(client)
-    )
+
+    mp_spec_targets = mp.spec.targets if mp.spec.targets is not None else []
```

Comment:
```suggestion
    mp_spec_targets = mp.spec.targets or []
```
is it the same, isn't it?

Reply:
Almost, but its more implicit than original condition

Reply:
OK, I have understood.